packages/suite/src/views/wallet/transactions/components/TransactionList/index.tsx

```diff
@@ -116,6 +116,8 @@ const TransactionList = ({ transactions, isLoading, account, ...props }: Props)
     const isRipple = account.networkType === 'ripple';
     const showPagination = isRipple ? shouldShowRipplePagination : total > 1;
 
+    console.log(showPagination, isRipple, size, total);
```

Comment:
remove log

Reply:
Ugh, keep forgetting these logs :sweat_smile: We should probably add that to our linting rules.

Reply:
Addressed (& squashed) in 5338b4a